extensions/apicurio-registry-avro/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,11 @@
+---
+artifact: ${project.groupId}:${project.artifactId}:${project.version}
+name: "Apicurio Registry - Avro"
```

Comment:
I wonder if we need to mention "registry". 
@EricWittmann what do you think?

Reply:
My understanding is that Apicurio is an umbrella term that covers Studio, Registry and possibly more.